ksqldb-execution/src/main/java/io/confluent/ksql/execution/expression/tree/DecimalLiteral.java

```diff
@@ -37,6 +38,7 @@ public DecimalLiteral(final Optional<NodeLocation> location, final BigDecimal va
     this.value = requireNonNull(value, "value");
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP")
   @Override
   public BigDecimal getValue() {
```

Comment:
Not sure how to make `BigDecimal` immutable... Not sure how to make a deep-copy.